### PR TITLE
Addresses #649; parent button for MODELLER

### PR DESCRIPTION
**Pull request by vikramalva**

### CODE REVIEW

File: modules/results/src/main/twirl/views/resultpanels/NGL3DStructure.scala.html [hunk 1]
```diff
@@ -13,7 +16,7 @@
 }
 @if(resultname == "samcc_PDB_AXES") {
     If you use SamCC on our Toolkit for your research, please cite:
-}
+}f
```
Comment (zy4) [inline modules/results/src/main/twirl/views/resultpanels/NGL3DStructure.scala.html]: typo

**Comment by vikramalva** [comment]
@zin that's a point. It is also not the case for HTMP presently.

File: modules/results/src/main/twirl/views/resultpanels/NGL3DStructure.scala.html [hunk 2]
```diff
@@ -1,6 +1,9 @@
 @(filepath : String, filename : String, jobID : String, resultname : String)()
 
-<a type="button" onclick="@{resultname}_download()" id="downloadAlignment">Download PDB file</a>
+<div class="colorOptsAlignment">
+    <a type="button" onclick="@{resultname}_download()" id="downloadAlignment"><span>Download PDB file</span></a>
+    <a type="button" class="goToParentJob" style="display: none">Go to Parent Job</a>
```
Comment (zy4) [inline modules/results/src/main/twirl/views/resultpanels/NGL3DStructure.scala.html]: missing semicolon: `display: none;`

**Comment by zy4** [comment]
We can implement it later. It isn't that hard, though.

**Pull request merged**